admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
+
+=== Builds on a Virtual Network are Failing
+
+If you are installing OpenShift using a virtual network (for example,
+OpenStack), and a build is failing, the node maximum transmission unit (MTU)
+might not be compatible with the network MTU (for example, eth0).
+
+For a build to complete successfully, the MTU of an SDN must be less than the
+eth0 network MTU in order to pass data to between node hosts.
+
```

Comment:
Yes, there used to be MTU problems.  But we have solved those with recent builds of OpenShift (3.1.1.6 and later) and they should no longer be an issue unless you are (a) manually configuring the MTU in the node config file, or (b) have somehow manually adjusted the MTU on the SDN network interfaces like lbr0, vovsbr, vlinuxbr, or br0.

Instead I think it's probably more useful to say that if you have changed your NICs MTU or told OpenShift to use a different NIC that has a different MTU, *and* you have specified the MTU manually in the config file, that you'll probably want to change the config file.